server/schedule/operator/operator.go

```diff
@@ -757,9 +836,34 @@ func removePeerSteps(cluster Cluster, region *core.RegionInfo, storeID uint64, f
 	return
 }
 
+// findNoLabelProperty finds the first store without given label property.
+func findNoLabelProperty(cluster Cluster, prop string, storeIDs []uint64) (int, uint64) {
+	for i, id := range storeIDs {
+		store := cluster.GetStore(id)
+		if store != nil {
+			if !cluster.CheckLabelProperty(prop, store.GetLabels()) {
+				return i, id
+			}
+		} else {
+			log.Debug("nil store", zap.Uint64("store-id", id))
+		}
+	}
+	return -1, 0
+}
+
+// transferLeaderToAnySteps returns the first suitable store to become region leader.
+//   Return an error if there is no suitable store.
+func transferLeaderToAnySteps(cluster Cluster, leaderID uint64, storeIDs []uint64) (OpKind, []OpStep, error) {
```

Comment:
This name confused me, `Any` do not reflect `first match`.

Reply:
Renamed to `transferLeaderToSuitableSteps`.